src/frontend/komponenter/Fagsak/Personlinje/Behandlingsmeny/EndreBehandlendeEnhet/EndreBehandlendeEnhet.tsx

```diff
@@ -61,80 +57,87 @@ const EndreBehandlendeEnhet: React.FC = () => {
             <Dropdown.Menu.List.Item onClick={() => settVisModal(true)}>
                 Endre behandlende enhet
             </Dropdown.Menu.List.Item>
-            {visModal && (
-                <UIModalWrapper
-                    modal={{
-                        actions: [
-                            <Button
-                                key={'avbryt'}
-                                size="small"
-                                variant="secondary"
-                                onClick={lukkBehandlendeEnhetModal}
-                                children={'Avbryt'}
-                            />,
-                            <Button
-                                key={'bekreft'}
-                                variant="primary"
-                                size="small"
-                                disabled={submitRessurs.status === RessursStatus.HENTER}
-                                onClick={() => {
-                                    if (åpenBehandling.status === RessursStatus.SUKSESS) {
-                                        endreEnhet(åpenBehandling.data.behandlingId);
-                                    }
-                                }}
-                                children={'Bekreft'}
-                                loading={submitRessurs.status === RessursStatus.HENTER}
-                            />,
-                        ],
-                        onClose: lukkBehandlendeEnhetModal,
-                        lukkKnapp: true,
-                        tittel: 'Endre enhet for denne behandlingen',
-                        visModal: visModal && !erBehandlingAvsluttet,
+            {visModal && !erBehandlingAvsluttet && (
```

Comment:
Foreslår å flytte sjekke om behandligen er avsluttet til det stedet som velger om hele komponenten skal rendres eller ikke. For nå blir det litt rart at menyvalget finnes, men hvis du klikker på det skjer det ingenting 😅

Reply:
Men det er forsåvidt noe som passer i en helt egen PR (altså å rydde opp i når de ulike valgene i menyen rendres), så her kan vi kanskje bare droppe å sjekke det?

Reply:
Jaa, fant noen andre rariteter i den menyen også. Tenker at det ikke er en del av denne PR'en å rydde opp i den logikken. Nå har jeg bare gjenbrukt akkurat logikken som var fra før 🤔